go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/researchnow/go-samplifyapi-client
 
-go 1.13
+go 1.14
```

Comment:
Is this upgrade needed?

Reply:
Since this is a public repo, let's keep it 1.13

Reply:
Done.